checkov/terraform/checks/graph_checks/oci/AdministratorUserAssociatedWithAPIKey.yaml

```diff
@@ -0,0 +1,65 @@
+metadata:
+  id: "PLACEHOLDER"
```

Comment:
@Eliran-Turgeman doesn't sound like the correct ID to me 😆

Reply:
Changed to CKV2_OCI_1